app/controllers/users_controller.rb

```diff
@@ -1,4 +1,12 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
```

Comment:
khi dùng filter cho >3 actions thì em dùng `except`
vd:
```suggestion
  before_action :logged_in_user, except: %i(show new create)
```